src/views/newUserOnboarding/index.js

```diff
@@ -21,13 +20,27 @@ import {
   StickyRow,
   ContinueButton,
 } from './style';
+import type { UserInfoType } from '../../../shared/graphql/fragments/user/userInfo';
```

Comment:
You can import directly from `shared/..` - no need for relative paths :)

Reply:
@brianlovin thanks, i fixed it!
In relation to this, flow type information is valuable to know **what kind of data?** it deals with people who are not familiar with codebase like me, so I'd like to continue to contribute!:smile: (cc: @mxstbr )

e.g. Replacing a simple `Object` type to a more appropriate GraqhQL Fragment types.

Reply:
@mxstbr thanks bunch of review, and appreciate.
i think flow is not affect in runtime code, so we can improve aggressively when you thought it!

i like James Kyle's thought, direction, so i reminder myself to not become **dogmatic typeckeck man** , for more human friendly information😀
https://discuss.reactjs.org/t/if-typescript-is-so-great-how-come-all-notable-reactjs-projects-use-babel/4887/2

also flow provide `// @noflow` escape hatch(also you know `$FlowFixMe`), we can use these 
as well for do not sacrifice productivity👩‍💻
https://github.com/facebook/flow/issues/1321#issuecomment-181541953

Reply:
Yep, you get it—that's exactly how we use it! :+1: Any improvements to our flow types is very welcome.